### PR TITLE
Fix: Remove unused views

### PR DESCRIPTION
**Pull request by BackEndTea**
This PR

* [x] Deletes twig templates that are no longer in use

### CODE REVIEW

File: templates/layouts/default.twig [hunk 1]
```diff
@@ -1,65 +0,0 @@
-<!DOCTYPE html>
```
Comment (localheinz) [inline templates/layouts/default.twig]: Not sure about this one, @BackEndTea:

```
$ git grep "layouts/default.twig"
```

yields

```
resources/views/dashboard.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/401.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/403.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/404.twig:1:{% extends "layouts/default.twig" %}
resources/views/error/500.twig:1:{% extends "layouts/default.twig" %}
resources/views/home.twig:1:{% extends "layouts/default.twig" %}
resources/views/ideas.twig:1:{% extends "layouts/default.twig" %}
resources/views/layouts/admin.twig:1:{% extends "layouts/default.twig" %}
resources/views/layouts/reviewer.twig:1:{% extends "layouts/default.twig" %}
resources/views/package.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/create.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/edit.twig:1:{% extends "layouts/default.twig" %}
resources/views/talk/view.twig:1:{% extends "layouts/default.twig" %}
resources/views/user/change_password.twig:1:{% extends "layouts/default.twig" %}
resources/views/user/edit.twig:1:{% extends "layouts/default.twig" %}
```

File: templates/layouts/dashboard.twig [hunk 2]
```diff
@@ -1,74 +0,0 @@
-<!DOCTYPE html>
```
Comment (localheinz) [inline templates/layouts/dashboard.twig]: Running

```
$ git grep "layouts/dashboard.twig"
```

yields

```
resources/views/oauth/authorize.twig:1:{% extends "layouts/dashboard.twig" %}
resources/views/user/create.twig:1:{% extends "layouts/dashboard.twig" %}
resources/views/user/reset_password.twig:1:{% extends "layouts/dashboard.twig" %}
```

but maybe these aren't used either?

File: templates/layouts/dashboard-nowrap.twig [hunk 3]
```diff
@@ -1,61 +0,0 @@
-<!DOCTYPE html>
```
Comment (localheinz) [inline templates/layouts/dashboard-nowrap.twig]: 👍

File: templates/layouts/default.twig (hunk 1 above)
Comment (BackEndTea) [reply templates/layouts/default.twig]: Those link towards ```resources/layouts/default.twig``` not ```templates/layouts/default.twig```

File: templates/layouts/admin.twig [hunk 4]
```diff
@@ -1,86 +0,0 @@
-<!DOCTYPE html>
```
Comment (localheinz) [inline templates/layouts/admin.twig]: Running

```
$ git grep "layouts/admin.twig"
```

yields

```
resources/views/admin/admins/index.twig:1:{% extends "layouts/admin.twig" %}
resources/views/admin/index.twig:1:{% extends "layouts/admin.twig" %}
resources/views/admin/speaker/index.twig:1:{% extends "layouts/admin.twig" %}
resources/views/admin/speaker/view.twig:1:{% extends "layouts/admin.twig" %}
resources/views/admin/talks/index.twig:1:{% extends "layouts/admin.twig" %}
resources/views/admin/talks/view.twig:1:{% extends "layouts/admin.twig" %}
```

but maybe these aren't used either?

File: resources/views/admin/talks/_table.twig [hunk 5]
```diff
@@ -1,33 +0,0 @@
-<table>
```
Comment (localheinz) [inline resources/views/admin/talks/_table.twig]: 👍

**Comment by chartjes** [comment]
@BackEndTea @localheinz What is the status of this? Can we merge it or not

**Comment by BackEndTea** [comment]
From what i found this can be safely merged. Nowhere in the application are any references  to the old views location.

**Pull request merged**